docs/books/modules/user-guide/how-routers-route-messages.adoc

```diff
@@ -64,6 +64,12 @@ Flow control is "real" in that credits flow across the link route from the recei
 * Server-side selectors
 +
 With a link route, consumers can provide server-side selectors for broker subscriptions.
+
+* Modified delivery states
```

Comment:
I would use something like 'consumer specific negative acknowledgement' as the 'title' here. Modified states in general are not an issue it is the consumer scoped undeliverable-here flag that is the issue.